src/Schema/Extensions/GraphQLExtension.php

```diff
@@ -10,19 +10,18 @@
     /**
      * Manipulate the schema.
      *
-     * @param DocumentAST $documentAST
-     *
-     * @return DocumentAST
+     * @param  \Nuwave\Lighthouse\Schema\AST\DocumentAST  $documentAST
+     * @return \Nuwave\Lighthouse\Schema\AST\DocumentAST
      */
-    public function manipulateSchema(DocumentAST $documentAST)
+    public function manipulateSchema(DocumentAST $documentAST): DocumentAST
```

Comment:
This is an abstract class that is meant to be extended by the user so let's remove the return types on this class.

Reply:
Done